lib/racecar/cli.rb

```diff
@@ -26,7 +26,7 @@ def run
       RailsConfigFileLoader.load!
 
       if File.exist?("config/racecar.rb")
-        require "config/racecar"
+        require "./config/racecar"
```

Comment:
Is this change relevant?

Reply:
without this change I wasn't able to run my forked version. it keeps searching the config file under the original gem...